docs/Install/Starting_SLM-UI_manually.md

```diff
@@ -0,0 +1,32 @@
+# Starting SLM-UI manually 
+
+SLM-UI containers are automatically started by the SLM-UI installer. But, if it is not running on the host, users need to manually start it using the following commands.
```

Comment:
Is there a reference to this new file ?

We can include this as a point in  "Running Swarm Learning examples using SLM-UI", right at the start.